wagtailio/project_styleguide/templates/patterns/components/streamfields/features/feature_block.html

```diff
@@ -0,0 +1,7 @@
+{% load static wagtailcore_tags %}
+
+<div>
+    <h4>{{ value.heading }}</h4>
+    <div class="rich-text">{{ value.description|richtext }}</div>
+    <a href="{{ value.link }}">View docs</a>
```

Comment:
I could be wrong, but do you think we should allow this to be editable?

Reply:
As indicted in https://github.com/wagtail/wagtail.org/pull/212#discussion_r946862957, I think it makes sense to allow for flexibility for editors, while still providing a default / fallback value (**View docs**)

```suggestion
    {% if value.link %}
        <a href="{{ value.link }}">
            {% if value.link_title %}{{ value.link_title }}{% else %}View docs{% endif %}
        </a>
    {% endif %}
```

Reply:
implemented in 1a36be7

Reply:
hmm, I'm not sure if I agree, what happens if someone does not have anything meaningful to link to for a single point?

Reply:
The more flexible solution would be to either, supply a link title or be blank?

Reply:
> hmm, I'm not sure if I agree, what happens if someone does not have anything meaningful to link to for a single point?

If a link is not provided, then we do not worry about the link title, and the anchor element is not rendered

Reply:
> The more flexible solution would be to either, supply a link title or be blank?

What would be the best way to handle a situation where the link title is not specified? My thoughts: since we probably don't want to have something like `<a href="{{ value.link }}"></a>`, we could

1. `<a href="{{ value.link }}">{{ value.link }}</a>`  -- the problem is the link may be too long
2.  `<a href="{{ value.link }}">{{ value.link|cut:"https://"|cut:"http://"|cut:"www." }}</a>`
3. update context in **FeatureBlock** so that we have a "trimmed link" with no scheme and www subdomain, then use this as link title if one isn't provided